pages/experiments/index.vue

```diff
@@ -43,7 +43,7 @@ import { Component } from 'vue-property-decorator'
 import InnerNavigation from '~/components/menus/InnerNavigation.vue'
 import GatesHeader from '~/components/headers/GatesHeader.vue'
 import PageSection from '~/components/sections/PageSection.vue'
-import Cta from '~/components/ctas/Cta.vue'
+// import Cta from '~/components/ctas/Cta.vue'
```

Comment:
The reason why I commented this import is because the [line 11](https://github.com/qiskit-community/community.qiskit.org/blob/848d4c567ea26b8198caaf5640c759f16d23ce7d/pages/experiments/index.vue#L11) is commented as well. @delapuente , should I delete all these comments? What kind of value do they give?

Reply:
Yes, please, remove all the dead code.

Reply:
Deleted!